doc/user/content/manage/user-management/rbac-intro.md

```diff
@@ -0,0 +1,119 @@
+
+In Materialize, Role-Based Access Control allows organization administrators to:
+
+* determine which users have read or write permissions for specific objects
+
+* control how users interact with clusters by giving them different levels of access to
+resources
+
+* prevent accident operations from unauthorized users
+
+* isolate access to user-facing data from internal organization data
+
+Materialize object access is also dependant on cluster privileges.
+Users who need access to an object like a schema or a database must also have
+the same level of access to the cluster.
+
+!(#TODO_Diagram)
+
+The next sections go over the concepts of authorization and authentication and
+the objects within Materialize.
+
+## Authentication vs. Authorization
+
+First, we need to make a distinction between authentication and authorization.
+Authentication dictates who can log in to a system while authorization
+determines what a user can access within a system.
+
+Authentication in Materialize is handled when you sign up or are invited to a
+Materialize organization. Your immediate privileges are determined by your
+invitation from an administrator or are assigned automatically by Materialize
+when you sign up.
+
+Authorization is determined in RBAC by your organization administrator. When you
+invite users to your Materialize organization, you have the option to give the
+user more elevated privileges in the Materialize administrator console.
+
+## RBAC Structure
+
+RBAC in practice is a group of roles with assigned privileges and attributes.
+You can assign specific users to roles or assign privileges to users to inherit
+from other roles.
+
+### Roles
+
+A role is a collection of permissions you can apply to users. Roles make it
+easier to assign or revoke privileges on Materialize objects. You can group
+users into specified roles with different levels of permissions and adjust those
+permissions to ensure they have the correct level of access to objects.
+
+### Role Attributes
+
+Role attributes are actions available to any role you create. Attributes are
+independent of any other object in Materialize and apply to the entire
+organization. You can edit these actions when you create the role:
+
+| Name            | Description                                                                     |
+|-----------------|---------------------------------------------------------------------------------|
+| `CREATEDB`      | Can create a database.                                                          |
+| `CREATEROLE`    | Can create, alter, drop, grant membership to, and revoke membership from roles. |
+| `INHERIT`       | Can inherit the privileges of roles that it is a member of. On by default.      |
+| `CREATECLUSTER` | Can create a cluster.                                                           |
+
+### Privileges
+
+Privileges are the actions or operations a role is allowed to perform. When you
```

Comment:
Maybe specify that they are actions or operations a role is allowed to perform with respect to a specific object.